src/configAPIClient.ts

```diff
@@ -113,7 +114,7 @@ export default function ConfigAPIClient(
     this: IConfigAPIClient,
     apiKey: string,
     config: SDKInitConfig,
-    mpInstance: MParticleWebSDK
+    mpInstance: IMParticleWebSDKInstance
```

Comment:
on line 88, i see `mpInstance: MParticleWebSDK`.  Should this now be `mpInstance: IMParticleWebSDKInstance` also?  If I do a global search for `mpInstance: MParticleWebSDK` there are several other occurrences.  Should these all be changed to `mpInstance: IMParticleWebSDKInstance`?

Reply:
I think I may have missed a few so I just did another pass to make sure.